src/runtime/components/nuxt-img.ts

```diff
@@ -79,6 +79,8 @@ export default defineComponent({
         link: [{
           rel: 'preload',
           as: 'image',
+          // @ts-expect-error might need to resolve upstream in unhead
```

Comment:
This is fixed in zhead [v2.1.2](https://github.com/harlan-zw/zhead/releases/tag/v2.1.2)

Reply:
In the context of this PR, what's the preferred resolution here? 

Wait for the new release of `@unhead/vue` to use `zhead@^2.1.2`? Or do you want me to update the comment here to something like `fixed in zhead v2.1.2, needs upstream adoption by @unhead/vue`?

Reply:
Good question, if you delete the lock file & node_modules it should update to this version and you can remove the comment altogether

Reply:
don't worry about refreshing lockfile in this pr; I can remove the comment when we next update deps....